yarl/quoting.py

```diff
@@ -1,7 +1,16 @@
+import os
 import re
+import sys
 from string import ascii_letters, ascii_lowercase, digits
 from typing import Optional, TYPE_CHECKING, cast
 
+
+NO_EXTENSIONS = bool(os.environ.get("YARL_NO_EXTENSIONS"))  # type: bool
```

Comment:
it's shorter:
```suggestion
NO_EXTENSIONS = bool(os.getenv("YARL_NO_EXTENSIONS"))  # type: bool
```

Reply:
Do you really care? I prefer dict-like `os.environ` over more narrow `os.getenv()` / `os.setenv()`.

Reply:
I like it more but I don't have strong opinions regarding this. Feel free to ignore this.